src/Data/List/Relation/Unary/AllPairs/Properties.agda

```diff
@@ -132,3 +134,13 @@ module _ {R : Rel A ℓ} {P : Pred A p} (P? : Decidable P) where
   filter⁺ {x ∷ xs} (x∉xs ∷ xs!) with does (P? x)
   ... | false = filter⁺ xs!
   ... | true  = All.filter⁺ P? x∉xs ∷ filter⁺ xs!
+
+------------------------------------------------------------------------
+-- pointwise
+
+module _ {R : Rel A ℓ}  where
+
+  pointwise⁺ : {xs : List (Maybe A)} → AllPairs (Pointwise R) xs → AllPairs R (catMaybes xs)
```

Comment:
The name for this should be `catMaybes⁺` rather than `pointwise⁺`, as we're introducing the relation `AllPairs` over it.

Reply:
I renamed it.